bokehjs/src/lib/models/plots/plot_canvas.ts

```diff
@@ -880,6 +882,8 @@ export class PlotView extends LayoutDOMView implements Renderable {
   }
 
   protected _actual_paint(): void {
+    logger.trace(`PlotView._actual_paint ${this.model.id} ${this._render_count} start`)
```

Comment:
I would suggest this:
```suggestion
    logger.trace(`${this.toString()}._actual_paint ${this._render_count} start`)
```
to get consistent output with the rest of bokehjs (this includes the actual class name and model ID).